src/Mod/Spreadsheet/App/Sheet.cpp

```diff
@@ -1007,6 +1007,7 @@ DocumentObjectExecReturn *Sheet::execute(void)
                 boost::topological_sort(graph, std::front_inserter(make_order));
             } catch (std::exception&) {

```

Comment:
Could be specific, for e.g. [`not_a_dag`](https://www.boost.org/doc/libs/1_82_0/libs/graph/doc/exception.html#not_a_dag).